app/layersproxymodel.h

```diff
@@ -18,7 +18,7 @@
 
 #include "layersmodel.h"
 
-enum ModelTypes
+enum LayerModelTypes
```

Comment:
I changed this because as I wanted to create enum ModelTypes for projectsModel, this enum had the same name. Thus I changed that to be more explanatory. I can also create namespace